benches/bloom_bench.rs

```diff
@@ -0,0 +1,64 @@
+use bytes::{BufMut, BytesMut};
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use slatedb::filter::{filter_hash, BloomFilterBuilder};
+fn bloom_filter_insert_benchmark(c: &mut Criterion) {
+    let mut builder = BloomFilterBuilder::new(10); // Set bits_per_key
+    let key_sz = std::mem::size_of::<u32>();
+
+    c.bench_function("bloom_filter insert 10000000 keys", |b| {
+        b.iter(|| {
+            for i in 0..10000000 {
```

Comment:
For the insert benchmark I don't think we need to insert so many keys. To understand key insert performance a single key would suffice. Criterion will take care of running the benchmarked code over many iterations to get representative data.